rpctest/rpc_harness_test.go

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2016 The decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+package rpctest
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/decred/dcrd/chaincfg"
+	"github.com/decred/dcrutil"
+)
+
+func TestSendFrom(t *testing.T) {
+	// Create a new test instance.
+	nodeTest, err := New(&chaincfg.SimNetParams, nil, nil)
```

Comment:
I suspect this (and `TestSendToAddress`) could make use of the instance setup in `TestMain` instead of needing to go through setup and teardown of everything again.